bin/rtd-docs

```diff
@@ -5,20 +5,29 @@ set -e
 set -x
 
 if [ "${READTHEDOCS_PROJECT}" = "warehouse" ]; then
+  if [ "$READTHEDOCS_VERSION_TYPE" = "external" ] && git diff --quiet origin/main -- docs/dev requirements/docs-dev.txt .readthedocs.yaml; then
+    exit 183;
```

Comment:
Would adding more context to why we exit be helpful, or does RTD response hook provide enough context back to GitHub as to that it was skipped, and their output don't need further logging?

```suggestion
    echo "Nothing relevant changed, skipping build...
    exit 183;
```